### PR TITLE
CloudFormation migration

### PR DESCRIPTION
**Pull request by jesdynf**
- [x] moves VPC, RDS, EFS, Redis, CloudWatch into one CloudFormation script
 - [ ] test
 - [x] updates documentation
 - [ ] review
 
* beanstalk work in later branch

### CODE REVIEW

**Comment by ghost** [comment]
@MatthewVita Is there anyway to preserve the original, step by step guide? I kind of liked it, it was like a good tutorial almost. I had actually been sending it to people I wanted to maybe join the group / get good experience.  

@jesdynf Can you put your code in a separate folder, maybe `V1.5-Beta-CF`? I can change the name of the original folder to `V1.0-Beta-Console`

**Comment by jesdynf** [comment]
@danielehrlich The answer to your question is apparently "no", but maybe you can? I modified my branch to put all my things in a new directory, but I'm pretty sure that applying this branch would delete the original v1 beta you want to preserve. Maybe @MatthewVita can untangle this?

**Comment by MatthewVita** [comment]
Don't worry, I'll be able to work my git magic when the time comes to check in @jesdynf's work. I'll be able to preserve the manual guide and have the automated stuff there

**Comment by MatthewVita** [comment]
Also, don't worry too much about folder names or release versions at the moment. We'll figure that out when the CF is done and fully tested

**Comment by MatthewVita** [comment]
One nice feature (or bug, if you read http://www.jayhuang.org/blog/pushing-code-to-github-as-linus-torvalds/) of git is that I can rearrange your branch but keep you as the author :)

File: v1-CloudFormation/assets/cf/OpenEMR.json [hunk 1]
```diff
@@ -15,11 +15,23 @@
       "MinLength" : "10",
       "MaxLength" : "41",
       "AllowedPattern" : "^[a-zA-Z0-9]*$"
+    },
+    "TimeZone": {
+      "Type": "String",
+      "Default": "America\\\\/Chicago",
```
Comment (MatthewVita) [inline v1-CloudFormation/assets/cf/OpenEMR.json]: with the recent commit, can the escape be removed?

File: v1-CloudFormation/assets/cf/OpenEMR.json [hunk 2]
```diff
@@ -403,35 +415,64 @@
...
+          "S3Key" : { "Fn::FindInMap" : [ "RegionData", { "Ref" : "AWS::Region" }, "ApplicationSource"]}
+        }
+      }
     },
 
+    "EBEnvironment" : {
+     "Type" : "AWS::ElasticBeanstalk::Environment",
+     "Properties" : {
+       "ApplicationName" : { "Ref" : "EBApplication" },
+        "Description" :  "OpenEMR v5.0.0, cloud deployment 001",
```
Comment (MatthewVita) [inline v1-CloudFormation/assets/cf/OpenEMR.json]: Should be `OpenEMR v5.0.0-2`

File: v1-CloudFormation/chapters/03-Application-Servers.md [hunk 3]
```diff
@@ -2,98 +2,17 @@ _[< previous chapter](02-Application-Backbone.md) | [next chapter >](04-Secure-D
...
+TODO 1. (Explain how to get to the stack output)
+2. Go through each step of the signup wizard using the MySQL credentials noted in previous steps. Make sure to enter a [strong password](https://www.random.org/passwords/?num=1&len=16&format=html&rnd=new) for the initial user and record it in a safe place. Note that the first step of the wizard will take a few minutes. Although the page will be white and not have any loading indicators, please do not attempt to refresh the page or resubmit the request.
 
 ### Post-install security update
 
+TODO (this can't happen unless VPN is configured)
+
 1. In the AWS Management Console, click **EC2** and then click **Instances** in the left hand pane.
-2. Clickbox the running **your_practice** instance and note the **Public DNS (IPv4)** in the bottom pane.
+2. Clickbox the running **your_practice** instance and note the **Private DNS (IPv4)** in the bottom pane.
```
Comment (MatthewVita) [inline v1-CloudFormation/chapters/03-Application-Servers.md]: Good catch. I'm going to update the manual guide with this correction.

File: v1-CloudFormation/chapters/02-Application-Backbone.md [hunk 4]
```diff
@@ -14,6 +14,7 @@ _CloudFormation is used for setting up OpenEMR._
 6. Define the parameters of the application stack here.
 ..* For Stack name, answer "OpenEMR".
 ..* For EC2KeyPair, select the EC2 key you created in chapter 1 from the dropdown.
+..* For Time Zone, set your timezone, putting two backslashes behind the / as shown in the example.
```
Comment (MatthewVita) [inline v1-CloudFormation/chapters/02-Application-Backbone.md]: :)

File: v1-CloudFormation/assets/cf/OpenEMR.json (hunk 2 above)
Comment (jesdynf) [reply v1-CloudFormation/assets/cf/OpenEMR.json]: There's filenames involved and buckets, but I'll clean it up.

**Comment by nkduodu** [comment]
Hello is openemr getting an inpatient module?

**Pull request closed without merging**